youtube_dl/extractor/cbc.py

```diff
@@ -165,12 +166,11 @@ class CBCPlayerIE(InfoExtractor):
             'uploader': 'CBCC-NEW',
         },
     }, {
-        # available only when we add `formats=MPEG4,FLV,MP3` to theplatform url
```

Comment:
Even if it's no longer true? This URL gives me only `http-621` format both with and without `formats=MPEG4,FLV,MP3`. Also tested using some Canadian proxy and results are the same.